aries-site/src/examples/templates/wizard/WizardAnatomy.js

```diff
@@ -0,0 +1,219 @@
+      <Annotation
+        alignSelf="center"
+        id="1a"
+        target="1a"
+        gridArea="annotation-1a"
+      />
+      <Annotation id="1b" target="1b" gridArea="annotation-1b" />
+      <Annotation id="1c" target="1c" gridArea="annotation-1c" />
+      <Annotation id="2b" target="2b" gridArea="annotation-2b" />
+      <Annotation id="2a" target="2a" gridArea="annotation-2a" />
+      <Annotation id="2c" target="2c" gridArea="annotation-2c" />
+      <Annotation
+        id="3"
+        target="3"
+        gridArea="annotation-3"
+        alignSelf="center"
+      />
+      <Annotation
+        id="4"
+        target="4"
+        gridArea="annotation-4"
+        alignSelf="center"
+      />
+      <Annotation
+        id="4a"
+        target="4a"
+        gridArea="annotation-4a"
+        alignSelf="center"
+      />
+    </Grid>
+    <Diagram connections={connections} />
+  </Stack>
+);
+
+WizardAnatomy.propTypes = {
+  containerRef: PropTypes.node,
+};
+
+const defaultFormValues = {
+  'firstname-validation': '',
+  'lastname-validation': '',
+  'text-input-validation': '',
+  'radio-button-group-validation': '',
+};
+
+// placeholders so anatomy diagram shows second step
+const steps = [
+  {
+    title: 'Placeholder',
+  },
+  {
+    description: `Step 2 description. Even though this step is single-column, 
+    the width of the footer should be consistent across all steps.`,
+    inputs: <StepOne />,
+    title: 'Step 2 title',
+  },
+  {
+    title: 'Placeholder',
+  },
+];
+
+const WizardExample = ({ containerRef, ...rest }) => {
+  const breakpoint = useContext(ResponsiveContext);
+  const theme = useContext(ThemeContext);
+
+  const numberColumns = 2;
+  const width = getWidth(numberColumns, theme, breakpoint);
+
+  const [activeIndex, setActiveIndex] = useState(1);
+  // for readability, this is used to display numeric value of step on screen,
+  // such as step 1 of 3. it will always be one more than the active array index
+  const [activeStep, setActiveStep] = useState(activeIndex + 1);
+
+  // store form values in state so they persist
+  // when user goes back a step
+  const [formValues, setFormValues] = useState(defaultFormValues);
+
+  // controls state of cancel layer
+  const [open, setOpen] = useState(false);
+
+  useEffect(() => {
+    setActiveStep(activeIndex + 1);
+  }, [activeIndex]);
+
+  const ref = useRef();
+
+  const contextValue = useMemo(
+    () => ({
+      activeIndex,
+      id: 'anatomy',
+      defaultFormValues,
+      setActiveIndex,
+      activeStep,
+      setActiveStep,
+      valid: true,
+      steps,
+      formValues,
+      setFormValues,
+      wizardTitle: 'Wizard Title',
+      width,
+    }),
+    [activeIndex, activeStep, formValues, width],
+  );
+
+  return (
+    <WizardContext.Provider value={contextValue}>
+      <Box
+        ref={ref}
+        background="background-front"
+        height="min-content"
```

Comment:
The header unintentionally aligns with the wizard content. I think we want the header to span full width.

```suggestion
        height="min-content"
        fill="horizontal"
```

![Screen Shot 2022-11-08 at 8 41 10 AM](https://user-images.githubusercontent.com/1756948/200609963-7e147aae-2c3e-408b-9b2c-c9490f3317b7.png)